### ISSUE TITLE
Add Bowdler spurs

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10531]
**Preferred term label:**
Bowdler spurs

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
Transverse long bone midshaft spurs or osteochondral projections typically occuring in the fibulae and less commonly in the forearms. PMID: 15782022

**Parent term (use hpo.jax.org/app)**
Abnormal long bone morphology HP:0011314

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hypophosphatasia (OMIM 146300, 241500, 251510), PMC6611735

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000873
- label: Bowdler spurs
- parents: [Abnormal long bone morphology (HP:0011314)](https://hpo.jax.org/app/browse/term/HP:0011314)
- pmid: PMID:15782022
- def: Transverse long bone midshaft spurs or osteochondral projections typically occuring in the fibulae and less commonly in the forearms.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10531
- synonyms:
